thoth/storages/graph/models.py

```diff
@@ -109,6 +109,7 @@ class PythonPackageLicense(Base, BaseExtension):
 
     # Relations
     python_package_version = relationship("PythonPackageVersion", back_populates="licenses")
+    __table_args__ = (UniqueConstraint("license_name", "license_identifier", "license_version"),)
```

Comment:
Could we not have a unique constraint on license_identifier only ?
If I'm not mistaken, this allows duplicates license_identifier for example, as long as the two other fields are not identical.